fdbserver/worker.actor.cpp

```diff
@@ -110,10 +110,14 @@ ACTOR Future<Void> forwardError( PromiseStream<ErrorInfo> errors,
 	}
 }
 
-ACTOR Future<Void> handleIOErrors( Future<Void> actor, IClosable* store, UID id, Future<Void> onClosed = Void() ) {
+ACTOR Future<Void> handleIOErrors( Future<Void> actor, IClosable* store, UID id, Future<Void> onClosed = Void(), bool ignoreReboots = false) {
 	choose {
```

Comment:
The ignoreReboots flag is unnecessary because please_reboot should now always ignore on closed